recipe/meta.yaml

```diff
@@ -52,7 +52,6 @@ build:
     - python *                               # [megabuild]
     - numpy *                                # [megabuild]
   skip: true  # [win]
-  skip: true  # [cuda_compiler_version != "None" and linux64 and blas_impl != "mkl"]
```

Comment:
since that commit came after the rerender, you didn't actually create any new jobs?

Also, for the time we're debugging this, you might want to inverse this condition (only build CUDA+non-MKL) to save CI resources.

Reply:
This doesn't affect available jobs — they were always there, they just terminate without building or producing any artifacts. And yes, that confused me too, so I've checked.

Reply:
Yeah, what happens is that some more complicated skip conditions don't get detected correctly by smithy (or are hard to specify so that this happens), which I guess is what happened here.